Extension/src/LanguageServer/client.ts

```diff
@@ -2387,10 +2390,47 @@ export class DefaultClient implements Client {
         return this.requestWhenReady(() => this.languageClient.sendRequest(SwitchHeaderSourceRequest, params));
     }
 
+    private async updateActiveDocumentTextOptions(): Promise<void> {
+        const editor: vscode.TextEditor | undefined = vscode.window.activeTextEditor;
+        if (editor
+            && editor.document
+            && editor.document.uri.scheme === "file"
```

Comment:
You could remove `editor && editor.document` if you use `editor?.document?.uri.scheme`